packages/types/src/interfaces/span.ts

```diff
@@ -80,6 +80,12 @@ export interface NodeSpan {
    */
   setError(error: Error): this
 
+  /**
+   * @deprecated since NodeJS version 2.1.0
```

Comment:
```suggestion
   * @deprecated since Node.js version 2.1.0
```

Looks like the correct spelling nowadays.